src/Manifest/ManifestManager.php

```diff
@@ -165,6 +159,36 @@ private function loadManifest(string $fileName, string $baseDir): array
         }
 
         $decoder = new JsonEncoder();
-        return $decoder->decode(file_get_contents(self::getManifestFilename($fileName)), JsonEncoder::FORMAT);
+        return $decoder->decode(file_get_contents($this->getManifestFilename($fileName)), JsonEncoder::FORMAT);
+    }
+
+    /**
+     * @param string $manifestAbsolutePath
+     * @param mixed[] $manifestContents
+     */
+    private function internalWriteManifest(string $manifestAbsolutePath, array $manifestContents): void
```

Comment:
Len jedna kozmeticka ... niekde to volas $options a niekde $manifestContents -> dal by som to na $options

Reply:
Protože někde to jsou option, ale tady už je to nějaké obecné pole, které může potenciálně přijít odkudkoli.